toolchain/sem_ir/typed_insts.h

```diff
@@ -1059,6 +1059,25 @@ struct PointerType {
   TypeId pointee_id;
 };
 
+// An instruction that indicates a type is required to be complete. This is only
+// created for generic types and produces a witness that the type is complete.
```

Comment:
```suggestion
// Indicates a type is required to be complete. This is only created for
// generic types and produces a witness that the type is complete.
```

I think we assume "An instruction" in this file.

Reply:
Switched to "Requires a type to be complete." I think that more directly describes the intended effect of this instruction.